Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -1336,29 +1336,20 @@ class RenderNodeTranslatorTests: XCTestCase {
         )
         let (bundle, context) = try await loadBundle(catalog: catalog)
 
-        func renderNodeArticleFromReferencePath(
-            referencePath: String
-        ) throws -> RenderNode {
-            let reference = ResolvedTopicReference(bundleID: bundle.id, path: referencePath, sourceLanguage: .swift)
-            let symbol = try XCTUnwrap(context.entity(with: reference).semantic as? Article)
-            var translator = RenderNodeTranslator(context: context, identifier: reference)
-            return try XCTUnwrap(translator.visitArticle(symbol) as? RenderNode)
-        }
-        
         // Assert that articles that curates any symbol gets 'API Collection' assigned as the eyebrow title.
-        var renderNode = try renderNodeArticleFromReferencePath(referencePath: "/documentation/unit-test/APICollection")
+        var renderNode = try renderNodeArticleFromReferencePath(context: context, bundle: bundle, referencePath: "/documentation/unit-test/APICollection")
```

Comment:
It should never be necessary to pass both a context and its inputs. Can you update this test helper to remove the `bundle` parameter?